stairway/src/test/java/bio/terra/stairway/impl/FilterTest.java

```diff
@@ -217,7 +217,7 @@ public void filterForm1OrderTest() throws Exception {
         "SELECT F.flightid, F.stairway_id, F.submit_time, F.completed_time,"
             + " F.output_parameters, F.status, F.serialized_exception, F.class_name"
             + " FROM flight F"
-            + " WHERE F.submit_time > :pagetoken"
+            + " WHERE F.submit_time < :pagetoken"
```

Comment:
So this was broken before? If so, it makes me wonder about how much value we get from asserting against string literals.

Reply:
Yeah, I think I see the value as a sanity check around the sql that gets built.  It's not enough of a test on its own.  The test that I added in the other class is the right place to catch the error that I'm fixing here.